subsys/zbus/Kconfig

```diff
@@ -18,6 +18,36 @@ config ZBUS_CHANNEL_NAME
 config ZBUS_OBSERVER_NAME
 	bool "Observer name field"
 
+config ZBUS_MSG_SUBSCRIBER
+	select NET_BUF
+	bool "Message subscribers will receive all messages in sequence."
+
+if ZBUS_MSG_SUBSCRIBER
+
+choice
+	prompt "ZBus msg_subscribers network buffer allocation"
+
+config ZBUS_MSG_SUBSCRIBER_NET_BUF_DYNAMIC
+	bool "Use heap to allocate network buffers data"
+
+config ZBUS_MSG_SUBSCRIBER_NET_BUF_STATIC
+	bool "Use fixed data size for network buffers pool"
```

Comment:
I don't think these should be referring to "network". I understand this is based on net_buf, but this sounds like a stale comment and there's no point tying the option name to the internal implementation. Drop the network/NET references, just call them subscriber buffers or something like that.